src/test/frontend/deploy/createnamespace_controller_test.js

```diff
@@ -22,7 +22,7 @@ describe('Deploy controller', () => {
     angular.mock.module(deployModule.name);
 
     angular.mock.inject(($mdDialog, $resource, $log) => {
-      ctrl = new NamespaceDialogController($mdDialog, $resource, $log, []);
+      ctrl = new NamespaceDialogController($mdDialog, $resource, $log, null, []);
```

Comment:
If youre already touching this. Use $controller service to create controllers, its much easier than manual work. See angular documentation for more info.